test/CadlRanchProjects/type/property/nullable/src/Generated/Models/BytesProperty.cs

```diff
@@ -74,7 +74,7 @@ internal BytesProperty()
         }
 
         /// <summary> Required property. </summary>
-        public string RequiredProperty { get; }
+        public string RequiredProperty { get; set; }
```

Comment:
what is causing the generation change?

Reply:
The client.tsp is removed. https://github.com/microsoft/typespec/issues/5399

This change also causes some tests fail. I added the ignore in those tests.

Reply:
could we create an issue to track these failures? later might our vendors could take a look

Reply:
https://github.com/Azure/autorest.csharp/issues/5205